cmd/spire-server/cli/run/run.go

```diff
@@ -385,6 +385,7 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 	}
 
 	logger, err := log.NewLogger(logOptions...)
+	sc.LaunchLogLevel, _ = logrus.ParseLevel(c.Server.LogLevel)
```

Comment:
in case it is unable to parse it will result that LaunchLogLevel will be Panic,
and user will not be noticed about a bad configuration, 
Can you capture error here and return a proper error?
and can you move this lines after error validation? 
error validation must be together to function that is returning error